Makefile

```diff
@@ -77,31 +77,31 @@ GOFUMPT        := $(GOBIN)/gofumpt
 $(GOLANGCILINT):
 	# To bump, simply change the version at the end to the desired version. The git sha here points to the newest commit
 	# of the install script verified by our team located here: https://github.com/golangci/golangci-lint/blob/master/install.sh
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/b90551cdf9c6214075f2a40d1b5595c6b41ffff0/install.sh | sh -s -- -b ${GOBIN} v1.32.2
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/b90551cdf9c6214075f2a40d1b5595c6b41ffff0/install.sh | sh -s -- -b ${GOBIN} v1.40.1
 
 $(GOIMPORTS):
-	$(GO) get -u golang.org/x/tools/cmd/goimports
+	$(GO) install golang.org/x/tools/cmd/goimports@latest
```

Comment:
You did this change because go get is deprecated? ("use of go get to build and install packages is deprecated" - https://tip.golang.org/doc/go1.16#modules)

Reply:
OK, i see the answer in the git commit message: https://github.com/oslokommune/okctl/pull/537/commits/4fc63af8bb1a20bf4efba6c1d6cfea1b610f01d6

Reply:
Not really. I understood it as a certain flag of `go get` was deprecated.

The difference (as I understood it) between `go get` and `go install` is that `go install` is more suited for CI since `go install` doesn't update `go.mod` and `go.sum`.